apps/settings/js/panels/hotspot_wifi_settings/panel.js

```diff
@@ -135,7 +134,8 @@ define(function(require) {
 
         // ensure SSID is set
         if (/^\s*$/.test(elements.tethering_ssid.value)) {
-          alert(_('SSIDCannotBeEmpty'));
+          navigator.mozL10n.formatValue('SSIDCannotBeEmpty').then(
+            msg => alert(msg));
```

Comment:
Here, with the delay of 5s, the panel returns to the main HotSpot settings and then shows the error dialog.

I can either mingle with the code to make it not return to the previous panel until the alert is shown or I could mingle with the code to make the "OK" button be disabled when the input is empty.
Would that work for you @evelynhung ?